src/main/resources/db/migration/V1__Create_table_for_planlagt_varsel.sql

```diff
@@ -0,0 +1,10 @@
+CREATE SEQUENCE PLANLAGT_VARSEL_ID_SEQ;
+
+CREATE TABLE PLANLAGT_VARSEL (
+  id                       CHAR(64)           PRIMARY KEY,
+  uuid                     VARCHAR(50)        NOT NULL UNIQUE,
```

Comment:
Kan `uuid`-feltet ha data typen `uuid`?

Og kanskje vi kan legge til `uuid-ossp`-modulen for å generere uuid'er? https://www.postgresql.org/docs/9.1/uuid-ossp.html